src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -6887,7 +6844,8 @@ namespace SketcherGui {
 
         virtual void deactivated(ViewProviderSketch *sketchgui)
         {
-            sketchgui->setAxisPickStyle(true);
+            Q_UNUSED(sketchgui);

```

Comment:
```suggestion
            (void) sketchgui;
```
I'm always a bit disturbed when using Qt macro when there is no strong link to Qt (not inside a QObject or other). But maybe that's personal taste only. ;)

Reply:
Q_UNUSED is pervasive in FreeCAD. I am not a fan, but it is the "Werner approved" way. 

Personally, I think we should start using the ```maybe_unused``` attribute (since c++17), as it is a language feature.

See:
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused

Reply:
> I'm always a bit disturbed when using Qt macro when there is no strong link to Qt (not inside a QObject or other).

Well, we can also define our own macro FC_UNUSED.

> Q_UNUSED is pervasive in FreeCAD. I am not a fan

I prefer it over (void) because IMO it's more readable and because of capital letters hardly to overlook.

> Personally, I think we should start using the maybe_unused attribute (since c++17), as it is a language feature.

That's an option, too.